src/rules/no-firestore-object-arrays.ts

```
@@ -176,8 +141,206 @@ Prefer:
+          break;
+        }
+        case AST_NODE_TYPES.TSInterfaceDeclaration: {
+          interfaceNames.add(n.id.name);
+          break;
+        }
+        case AST_NODE_TYPES.TSEnumDeclaration: {
+          enumNames.add(n.id.name);
+          break;
+        }
+        case AST_NODE_TYPES.ExportNamedDeclaration: {
+          if (n.declaration) visitNode(n.declaration as unknown as TSESTree.Node);
```

ESLint

[error] Insert `⏎···········`

(prettier/prettier)